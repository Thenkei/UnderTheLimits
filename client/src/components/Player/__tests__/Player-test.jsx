// import React from 'react';
// import renderer from 'react-test-renderer';

// import Player from '..';

it('renders correctly', () => {
  // const tree = renderer.create(
  //   <Player
  //     value={{
  //       name: 'BLBL',
  //       score: 120,
  //       isGameMaster: true,
  //     }}
  //   />,
  // ).toJSON();
  // expect(tree).toMatchSnapshot();
});
