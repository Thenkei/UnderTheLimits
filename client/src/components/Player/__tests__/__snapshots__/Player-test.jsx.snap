// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<dt>
  &gt;
  <span
    className="badge"
  >
    BLBL
  </span>
  <span
    className="badge"
  >
    120
     
    pt(s)
  </span>
</dt>
`;
